specification/search/resource-manager/Microsoft.Search/preview/2025-02-01-preview/examples/SearchCreateOrUpdateServiceDisableLocalAuth.json

```diff
@@ -53,7 +53,16 @@
           },
           "disableLocalAuth": true,
           "authOptions": null,
-          "disabledDataExfiltrationOptions": []
+          "disabledDataExfiltrationOptions": [],
+          "upgradeAvailable": false,
+          "systemData": {
```

Comment:
Since we have location and sku as top level values, I wonder if we should elevate systemData to a top level node vs keeping it under the properties node.